src/www/components/contactpage/ContactPage.scss

```diff
@@ -0,0 +1,161 @@
+    margin-top: 2vh;
+    width: 100%;
+
+    form{
+        display: flex;
+        flex-direction: column;
+        margin-top: 2vw;
+        width: 100%;
+
+        * {
+            border: 1px solid $softBlack;
+            border-radius: 20px;
+            box-shadow: 1px 1px $softBlack;
+            font-family: $globalFonts;
+            font-size: 2vh;
+            height: 4vh;
+            margin-top: 1vh;
+            margin-left: 5rem;
+            margin-right: 5rem;
+            margin-bottom: 2vh;
+            outline: none;
+            padding: 0.75rem 1rem;
+            padding-left: 1.75rem;
+            width: 40vh;
+            word-break: break-all;
+        }
+
+        textarea{
+            height: 20vh;
+            resize: none;
+        }
+    }
+
+    .contact-form-block{
+        background-color: white;
+        border: 1px solid $softBlack;
+        border-radius: 2vh;
+        display: flex;
+        flex-direction: row;
+        height: 60vh;
+        margin-top: 2vh;
+        margin-bottom: 3vw;
+    }
+
+    .contact-form-title1{
+        color: black;
+        font-family: Poppins-Bold;
+        font-size: 3.1vw;
+        margin-top: 4.3vh;
+        line-height: 1.2;
+        text-align: center;
+        padding-bottom: 7px;
+    }
+
+    .contact-form-title2{
+        color: black;
+        font-family: $globalFonts;
+        font-size: 2vw;
+        line-height: 1.5;
+        text-align: center;
+    }
+
+    .form-image{
+        border-radius: 2vh;
+    }
+
+    .submit-btn{
+        align-items: center;
+        border-radius: 2vh;
+        cursor: pointer;
+        font-size: 3vh;
+        font-family: $globalFonts;
+        height: 6vh;
+        margin-left: 30%;
+        margin-top: 3vh;
+        padding-left: 1.5vh;
+        outline: none;
+        transition: 0.1s ease-in-out;
+        width: 40%;
+
+        &:hover {
+            background-color: $softBlack;
+          }
+    }
+
+    .title-block {
+        align-items: center;
+        display: flex;
+        margin-top: 2vh;
+        flex-direction: column;
+        top: 2vw;
+        color: black;
+        width: 100%;
+    }
+
+    .top-img{
+        position: absolute;
+        width: 100%;
+        z-index: -1;
+    }
+}
+
+@media only screen and(max-width: 1350px) {
+    .form-image{
+        display: none;
+    }
+}
+
+@media only screen and(max-width: 850px) {
+
+    .contact-form-block{
+        margin-top: 1vh;
+    }
+
+    .contact-form-title2{
+        display: none;
+    }
+
+  }
+
+@media only screen and(max-width: 600px) {
+    .container{
+        height: 83vh;
+        width: 100%;
+
+        .contact-form-block{
+            border: none;
+        }
+        
+        .contact-form{
+            > *{
+                margin-left: 0rem;
+                margin-right: 0rem;
+            }
+        }
+
+        .submit-btn{
+            margin-left: 30%;
+        }
+
+        .top-img{
+            display: none;
```

Comment:
If you fix the issues I mentioned before for the `width` & `padding` issues, then you don't have to hide the image.
The only adjustment needed is the padding for the title text, which needs to be decreased so that the text is still within the image.